ArduCopter/AP_Arming.cpp

```diff
@@ -44,7 +44,9 @@ bool AP_Arming_Copter::run_pre_arm_checks(bool display_failure)
     }
 
     // always check motors
-    if (!motor_checks(display_failure)) {
+    char failure_msg[50] = {};
```

Comment:
The number of character bytes in a MAVLINK message is 50 bytes. Therefore, a string that does not exceed 50 bytes is good.